src/bidiMapper/domains/context/BrowsingContextImpl.ts

```diff
@@ -848,11 +832,161 @@ export class BrowsingContextImpl {
     }
   }
 
+  async #parseRect(clip?: BrowsingContext.ClipRectangle) {
```

Comment:
add return type

Reply:
We don't use return types for private functions. The reason we do it for public functions is for contracting with external modules.